numscript.go

```diff
@@ -56,7 +56,20 @@ type (
 )
 
 func (p ParseResult) Run(ctx context.Context, vars VariablesMap, store Store) (ExecutionResult, InterpreterError) {
-	res, err := interpreter.RunProgram(ctx, p.parseResult.Value, vars, store)
+	return p.RunWithFeatureFlags(ctx, vars, store, nil)
+}
+
+func (p ParseResult) RunWithFeatureFlags(
```

Comment:
Here we add a new publicly exposed function which has a `featureFlags map[string]struct{}` param
We have a map with untyped keys instead of a struct so that removing feature flags is not a breaking change